### DISCUSSION TITLE
DisplayText guru help needed

### DISCUSSION THREAD

**Discussion by greymda** [arendst/Tasmota#21392] [Q&A] [answered]
Hello, i have a LilyGO TTGO T-Display 1.14in Display (T-Display) board https://templates.blakadder.com/lilygo_T-Display.html with IPS ST7789V 1.14” display using 135x240 resolution.

Can someone who understands DisplayText help me get this displayed on it, please?
![IMG_5820](https://github.com/arendst/Tasmota/assets/9991596/f63f8ae8-fe75-44b5-b37e-40cf7e29bb14)

Appreciate your help!

**Comment by dosipod** [comment, marked as answer]
@greymda  
I do not fully understand displaytext  yet nor did i have time to play with rules but this page has all the info 
you need ( more or less ) just try it out https://tasmota.github.io/docs/Displays/ 

Example is 
```
displaytext [zr120:120][x4y20][s2] PV which is  the same as displaytext [zr120:120x4y20s2] PV
```
i just broke it up so you could easily understand  how to fish  , i have also included my display config as you might need to change things
to fit your display 

![image](https://github.com/arendst/Tasmota/assets/87589371/a25a925f-a83b-417f-bb50-d16502741ad8)

In the picture with the full example  :
1) Clear display and draw the top left square with 120 width and 120 height
2) Change position to x and y 
3) Change text size to 2 and write the word PV
4) Change position to x and y 
5) Draw the top right  square with 120 width and 120 height
6) Change position to x and y and write the word Grid
7) Change position to x and y 
8) Draw bottom rectangle with 240 width and 120 height
9) Change position to x and y and write the word Today

So the full example is 
```
displaytext [zr120:120][x4y20][s2] PV[x120y0][r120:120][x121y20] Grid[x0y120][r240:120][x1y140] Today
```

Have a nice day

**Reply by greymda** [reply]
@dosipod apologies for late reply. THANK YOU so much for taking the time for this thorough response, will definitely give it a try!
